website/content/api-docs/system/internal-counters.mdx

```diff
@@ -89,24 +89,180 @@ $ curl \
+parameters.
+
+There are a few things to be known while using this API.
+
+- The activity information only accounts for activity that has happened for the
+latest contiguous months in the billing period. For example, if the billing
+period is from Jan 2022 to June 2022, and say the activity subsystem in Vault
+was not capturing data for the months Jan to March for any reason, the response
+will only include information for the months of May and June.
+
+- The response contains the total activity for the billing period and the
+attributions of the total activity against specific components in Vault. This
+helps in understanding the total activity better by knowing which components in
+Vault are leading to that top level activity count. First level of attribution
+breakdowns are by namespaces and months, under the `by_namespaces` and `months`
+JSON block respectively.
+
+```json
+{
+   "by_namespace":[],
+   "months":[],
+}
+```
+
+- `by_namespaces` breakdowns provide attributions of each namespace to the total
+activity count. These namespace attributions are further broken down by mount
+level attributions, wherein information can be found on attributions of each
+mount path within a given namespace to the overall activity of the namespace.
+
+```json
+{
+   "by_namespace":[
+      {
+         "namespace_id":"root",
+         "namespace_path":"",
+         "counts":{},
+         "mounts":[
+            {
+               "path":"auth/up1/",
+               "counts":{}
+            },
+            {
+               "path":"auth/up2/",
+               "counts":{}
+            }
+         ]
+      }
+   ]
+}
+```
+
+- `months` breakdowns provide attributions of each month to the total activity
+count. These month level attributions are further broken down into namespace and mount
+level attributions for each month.
+
+```json
+{
+   "months":[
+      {
+         "timestamp":"2021-05-01T00:00:00Z",
+         "counts":{},
+         "namespaces":[
+            {
+               "namespace_id":"root",
+               "namespace_path":"",
+               "counts":{},
+               "mounts":[
+                  {
+                     "path":"auth/up2/",
+                     "counts":{}
+                  },
+                  {
+                     "path":"auth/up1/",
+                     "counts":{}
+                  }
+               ]
+            },
+            {
+               "namespace_id":"s07UR",
+               "namespace_path":"ns1/",
+               "counts":{},
+               "mounts":[
+                  {
+                     "path":"auth/up1/",
+                     "counts":{}
+                  },
+                  {
+                     "path":"auth/up2/",
+                     "counts":{}
+                  }
+               ]
+            }
+         ],
+         "new_clients":{}
+      }
+   ],
+}
+```
+
+- Each entry in the `months` breakdown contains a `new_clients` block. Within a
```

Comment:
> "When a token is first used within a month, regardless of when it was created, that token is considered a new client for the month"

☝️ would be incorrect. "first used within a billing period" is what we want to say. The current diff states it this way.

> This is because in this section you're talking about months. The newness in the billing period is also true but not relevant, right? All the activity within a billing period is "new" because the newness counts reset at the beginning of a billing period?

You are right about the newness counts getting reset at the beginning of a billing period. It is relevant in the context of where it is placed. The previous bullet explains the `months` block. This one is detailing on how to read the `new_clients` within the `months` in particular. So, this paragraph explains the behavior for the first month, and then talks about how the new clients will be computed for the subsequent months.